packages/renderers/lib/src/request/request_item_renderer/widgets/value_renderer_list_tile.dart

```diff
@@ -11,7 +11,7 @@ class ValueRendererListTile extends StatefulWidget {
   final InputDecoration? decoration;
   final AttributeValue? initialValue;
   final ValueRendererController? controller;
-  final void Function({String? valueType, ValueRendererInputValue? inputValue, required bool isComplex}) onUpdateInput;
+  final Future<void> Function(String valueType, String? value)? onUpdateAttribute;
```

Comment:
```suggestion
  final void Function(String valueType, String? value)? onUpdateAttribute;
```

as it get's never awaited